### PR TITLE
docs(de): add german integrations guide

### PR DESCRIPTION
**Pull request by mikelgo**
This PR covers the german translation for https://docs.astro.build/de/guides/integrations-guide/

### CODE REVIEW

File: src/pages/de/guides/integrations-guide.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+<IntegrationsNav />
+
+## Autmatische Einrichtung von Integrationen
+
+Astro stellt ein `astro add` Kommando zur Verfügung, um das Einrichten von Integrationen zu automatisieren.
+
+:::Achtung
+Wir werden immer um Bestätigung bitten, bevor wir deine Dateien aktualisieren. Jedoch ist es empfehlenswert, ein Versionskontroll-Backup zu haben.
+:::
+Führe das `astro add` Kommando mit dem Paketmanager deiner Wahl aus und unser automatischer Integration-Wizard wird deine Konfigurationsdatei aktualisieren und alle notwendigen Abhängigkeiten installieren.
```
Comment (hippotastic) [inline src/pages/de/guides/integrations-guide.md:30]:
```suggestion

Führe das `astro add`-Kommando mit dem Paketmanager deiner Wahl aus und unser automatischer Integrations-Assistent wird deine Konfigurationsdatei aktualisieren und alle notwendigen Abhängigkeiten installieren.
```

File: src/pages/de/guides/integrations-guide.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn astro add react
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Es ist sogar möglich mehrere Integrationen gleichzeitig zu konfigurieren!
```
Comment (hippotastic) [inline src/pages/de/guides/integrations-guide.md:50]:
```suggestion
Es ist sogar möglich, mehrere Integrationen gleichzeitig zu konfigurieren!
```

File: src/pages/de/guides/integrations-guide.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn astro add react tailwind partytown
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+:::note[Abhängigkeiten von Integrationen managen]
```
Comment (hippotastic) [inline src/pages/de/guides/integrations-guide.md:70]:
```suggestion
:::note[Behandlung von Abhängigkeiten deiner Integrationen]
```

File: src/pages/de/guides/integrations-guide.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+Solltest du eine Warnung wie `Cannot find package '[package-name]'` erhalten, dann hat dein Paketmanager vermutlich die [peer dependencies](https://nodejs.org/en/blog/npm/peer-dependencies/) installiert. Um diese zu installieren, führe einfach `npm install [package-name]` aus.
```
Comment (hippotastic) [inline src/pages/de/guides/integrations-guide.md:71]:
```suggestion
Solltest du eine Warnung wie `Cannot find package '[package-name]'` nach dem Hinzufügen einer Integration erhalten, hat dein Paketmanager vermutlich nicht die zugehörigen [Peer Dependencies](https://nodejs.org/en/blog/npm/peer-dependencies/) installiert. Um diese fehlenden Abhängigkeiten zu installieren, führe einfach `npm install [package-name]` aus.
```

File: src/pages/de/guides/integrations-guide.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+:::
+
+## Integrationen nutzen
+
+Astro-Integrationen werden immer über die `integrations` Option in deiner `astro.config.mjs` Datei konfiguriert.
```
Comment (hippotastic) [inline src/pages/de/guides/integrations-guide.md:76]:
```suggestion
Astro-Integrationen werden immer über die `integrations`-Option in deiner `astro.config.mjs`-Datei konfiguriert.
```

File: src/pages/de/guides/integrations-guide.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+
+Es gibt drei Arten, um Integrationen zu importieren:
```
Comment (hippotastic) [inline src/pages/de/guides/integrations-guide.md:78]:
```suggestion
Es gibt drei übliche Wege, um eine Integration in dein Astro-Projekt zu importieren:
```

File: src/pages/de/guides/integrations-guide.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+import installedIntegration from '@astrojs/vue';
+import localIntegration from './my-integration.js';
+
+export default defineConfig({
+  integrations: [
+    // 1. Aus einem installiertem npm-Paket importieren
+    installedIntegration(),
+    // 2. Aus einer lokalen JS-Datei importieren
+    localIntegration(),
+    // 3. Ein Objekt
```
Comment (hippotastic) [inline src/pages/de/guides/integrations-guide.md:95]:
```suggestion
    // 3. Ein Inline-Objekt
```

File: src/pages/de/guides/integrations-guide.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+    {name: 'namespace:id', hooks: { /* ... */ }},
+  ]
+})
+```
+
+Schaue dir die [Integration-Schnittstelle](/en/reference/integrations-reference/) an, um mehr darüber zu erfahren, wie du deine eigenen Integrationen erstellen kannst.
```
Comment (hippotastic) [inline src/pages/de/guides/integrations-guide.md:101]:
```suggestion
Sieh dir die [Integrations-API](/de/reference/integrations-reference/) an, um mehr darüber zu erfahren, wie du deine eigenen Integrationen erstellen kannst.
```

We're already using the term API in various places, so it should be safe to not translate it here.

File: src/pages/de/guides/integrations-guide.md [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+
+```js
+integrations: [
+  // Beispiel: Argumente an eine Integration übergeben
+  sitemap({filter: true})
+]
+```
+### Integrationen aktivieren und deaktivieren
+
+Integrationen mit `falsy`-Werten, werden ignoriert. Dadurch können sie aktiviert oder deaktiviert werden und man muss sich keine Gedanken über hinterlassene `undefined`- und Boolean-Werte machen.
```
Comment (hippotastic) [inline src/pages/de/guides/integrations-guide.md:115]:
```suggestion
Integrationen mit `falsy`-Werten werden ignoriert. Dadurch können sie aktiviert oder deaktiviert werden und man muss sich keine Gedanken über hinterlassene `undefined`- und Boolean-Werte machen.
```

File: src/pages/de/guides/integrations-guide.md [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+```js
+integrations: [
+  // Beispiel: Keine Sitemap unter Windows erstellen
+  process.platform !== 'win32' && sitemap()
+]
+```
+
+## Mehr Integrationen entdecken
+
+Eine Vielzahl von Integrationen, die durch die Community entwickelt werden, können in dem [Astro-Integrationen Verzeichnis](https://astro.build/integrations/) gefunden werden. Beachte die Links für detaillierte Anleitungen.
```
Comment (hippotastic) [inline src/pages/de/guides/integrations-guide.md:126]:
```suggestion
Eine Vielzahl von Integrationen, die durch die Community entwickelt werden, können in [Astros Integrations-Verzeichnis](https://astro.build/integrations/) gefunden werden. Folge den dortigen Links, um detaillierte Anleitungen zu ihrer Benutzung und Konfiguration zu erhalten.
```

File: src/pages/de/guides/integrations-guide.md [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+
+## Eine eigene Integration erstellen
+
+Astro's Integrations-API ist durch Rollup und Vite inspiriert.
```
Comment (hippotastic) [inline src/pages/de/guides/integrations-guide.md:130]:
```suggestion
Astros Integrations-API ist durch Rollup und Vite inspiriert und wurde so gestaltet, dass sie sich für alle vertraut anfühlen sollte, die jemals ein Rollup- oder Vite-Plugin geschrieben haben.
```

File: src/pages/de/guides/integrations-guide.md [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+
+Schaue dir ie [Integration-Schnittstelle](/en/reference/integrations-reference/) an, um mehr darüber zu erfahren, wie du deine eigenen Integrationen erstellen kannst.
```
Comment (hippotastic) [inline src/pages/de/guides/integrations-guide.md:132]:
```suggestion
Sieh dir die [Integrations-API](/de/reference/integrations-reference/) an, um mehr darüber zu erfahren, was Integrationen leisten können und wie du deine eigenen Integrationen erstellen kannst.
```

**Review by hippotastic — COMMENTED** [review]
Thank you for contributing this translation! I only found some minor corrections (mostly grammar stuff). Please have a look :)

File: src/pages/de/guides/integrations-guide.md [hunk 13]
```diff
@@ -0,0 +1,132 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Integrationen nutzen
+i18nReady: true
+setup: |
+  import IntegrationsNav from '~/components/IntegrationsNav.astro';
+  import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+---
+
+**Astro-Integrationen** sind ein einfacher Weg neue Funktionalität mit wenigen Zeilen Code zu deinem Projekt hinzuzufügen. Du kannst eine Integration selbst schreiben, eine offizielle Integration nutzen oder Integrationen von der Community nutzen.
```
[the remark this answers is not in the document]
Comment (mikelgo) [reply src/pages/de/guides/integrations-guide.md:10]: applied

**Pull request merged**